Cargo.toml

```diff
@@ -15,11 +15,12 @@ categories = ["game-development"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-bevy = "0.7"
-bevy_rapier3d = { version = "0.15.0", features = ["debug-render"] }
+bevy = { version = "0.7", default-features = false, features = ["bevy_render"]}
```

Comment:
Is there a reason why you left `bevy_render` in? I don't believe it's necessary.

Reply:
@PROMETHIA-27 unfortunately yes because Rapier need it, but don't have it for `dim3`. I fixed it in https://github.com/dimforge/bevy_rapier/pull/214, but for now it's required if you use Rapier.